### DISCUSSION TITLE
能否对服务商使用"复制"功能?

### DISCUSSION THREAD

**Discussion by jiayaoO3O** [lobehub/lobehub#4039] [Ideas]
1. 功能现状 : 每一种类的服务商, 例如openai, 只能接入一次, 使用同一个key.
2. 现状困境 : 但是当我使用国内的中转时, 存在一种情况, 就是不同的模型会有不同的渠道, 需要使用不同的key, 使得当我想要用openai的模型和Claude3.5模型时, 需要手动进去设置界面切换key, 或者浏览器开隐私模式.
3. 希望得到改进 : 提供一个类似"复制"的功能, 使得同一个服务商, 可以输入不同的key, 配置不同的模型

**Comment by SpeedupMaster** [comment]
one api应该可以满足需求吧 [ref](https://github.com/songquanpeng/one-api)

**Reply by jiayaoO3O** [reply]
我买的那个代理转发就是不同渠道对应不同的key的，api地址都是同一个，但是不同的模型例如Claude和gpt-4o要走不同的key，所以还得来回切换

**Reply by SpeedupMaster** [reply]
这个可以用one api整合的，或者你直接在环境变量中分别配置Openai和Anthropic AI的api和url

**Reply by xiaokaiu** [reply]
https://api.001hao.com/这个可以用同一个key跑不同的

**Comment by arvinxx** [comment]
new ai provider 已经支持
